data/os/RedHat.yaml

```diff
@@ -0,0 +1,67 @@
+---
+## osfamily: RedHat
+
+yum::gpgkeys:
+  /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-6: # https://archive.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-6
```

Comment:
This will install the EPEL gpg key on osfamily RedHat even if epel is not enabled ? That seems undesirable.

Reply:
Actually, that is just the key in hieradata. It actually works similar to the "repos" and "managed_repos" logic. In order for a gpg key to be added to the system, a *repo* specifically referencing that key must be included in the `managed_repos` list. It does not simply "create_resources()" for each gpgkey in gpgkeys.  So, this specific key will not be added to the system, unless you have `epel` in manged_repos on an EL6 system (ceteris paribus).